tests/e2e/docs/vanilla_cluster_for_vcp2csi_migration_setup.md

```diff
@@ -0,0 +1,96 @@
+# E2E tests on Vanilla k8s
```

Comment:
Why are we duplicating https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/tests/e2e/docs/vanilla_cluster_setup.md?

Is it possible for you to highlight a needed change for in-tree to CSI migration test setup in the existing doc?

Reply:
I was not sure what other requirements might come, hence cloned it.

Reply:
> Is it possible for you to highlight a needed change for in-tree to CSI migration test setup in the existing doc?

+1

Reply:
changes merged into same file